gradle/plugins/src/main/kotlin/com.hedera.gradle.jpms-modules.gradle.kts

```diff
@@ -111,6 +111,24 @@ extraJavaModuleInfo {
     module("io.grpc:grpc-util", "io.grpc.util")
     module("io.grpc:grpc-protobuf", "io.grpc.protobuf")
     module("io.grpc:grpc-protobuf-lite", "io.grpc.protobuf.lite")
+    module("io.helidon.common:helidon-common", "io.helidon.common") {
+        exportAllPackages()
+        requireAllDefinedDependencies()
+        requires("java.logging")
+        patchRealModule()
+    }
+    module("io.helidon.webclient:helidon-webclient", "io.helidon.webclient") {
+        exportAllPackages()
+        requireAllDefinedDependencies()
+        requires("java.logging")
+        patchRealModule()
+    }
+    module("io.helidon.webclient:helidon-webclient-grpc", "io.helidon.webclient.grpc") {
+        exportAllPackages()
+        requireAllDefinedDependencies()
+        requires("java.logging")
+        patchRealModule()
+    }
```

Comment:
Are these really needed or is this some kind of leftover?
The modules already have a module descriptor. That's why `patchRealModule()` needed to be added.

Is this because of the `requires("java.logging")`? If so this should also be reported to the Helidon project to be fixed upstream.